shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1626,7 +1700,9 @@ - (UITextPosition*)closestPositionToPoint:(CGPoint)point {
              @"Expected a FlutterTextPosition for position (got %@).",
              [_selectedTextRange.start class]);
     NSUInteger currentIndex = ((FlutterTextPosition*)_selectedTextRange.start).index;
-    return [FlutterTextPosition positionWithIndex:currentIndex];
+    UITextStorageDirection currentAffinity =
+        ((FlutterTextPosition*)_selectedTextRange.start).affinity;
```

Comment:
why are we using the start's affinity? or is it irrelevant?

Reply:
This path is when we are missing selection rects, it is kind of a dummy answer anyways. I used the `start` affinity because the current practice was using the `start` index.